pkg/hintrunner/zero/zerohint_memcpy.go

```diff
@@ -3,9 +3,54 @@ package zero
 import (
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/core"
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+
 	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
 )
 
+func newMemcpyContinueCopyingHint(output hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "MemcpyContinueCopying",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> n -= 1
+			//> ids.continue_copying = 1 if n > 0 else 0
+
+			//> n-=1
+			n, err := ctx.ScopeManager.GetVariableValue("n")
+			if err != nil {
+				return err
+			}
+
+			n = n.(uint64) - 1
+			ctx.ScopeManager.AssignVariable("n", n)
+
+			//> ids.continue_copying = 1 if n > 0 else 0
+			continueCopyingAddr, err := output.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			var v memory.MemoryValue
+			if n.(uint64) > 0 {
```

Comment:
Isn't `n` already of the required type from the prev line `n = n.(uint64) - 1`

Reply:
when I tried to leave just `n > 0` it didn't work since it is treated as an interface and the comparison can't be done

Reply:
if `n` has already been typecasted to a `uint64` it should work

Reply:
@TAdev0 it says this: `invalid operation: n > 0 (operator > not defined on interface)`